docs/README.md

```diff
@@ -0,0 +1,44 @@
+# App design docs
+
+This provides an overview of how the calendar works.
+
+
+## Services
+
+![Service diagram](./services.png)
+
+DMS calendar communicates with:
+
+* MariaDB: holds persistent data (e.g. calendar events, attendees, etc.); MySQL-compatible database server
+* DMS Active Directory (LDAP): connects the calendar to DMS accounts used across DMS's other services; account / authentication server
+* Braintree: credit card processesing; third-party service
+* Twilio: sends SMS notifications; third-party service
+* Sparkpost: sends email notifications? third-party service
+
+
+## Webserver
```

Comment:
The webserver is actually just Apache and PHP-FPM. You're correct that `/index.php` is effectively what gets invoked, though all of the important stuff is in `/src`.

Reply:
The dev server uses HHVM though, right? Maybe I'll update this to describe production and development.

Regarding /index.php, I'll add more detail about what happens after /index.php is invoked.

Reply:
I don't think I've ever looked at the dev server, so I can't say for sure.

Reply:
OK - now that I know we use Apache / PHP-FPM in prod, I'll update the docs accordingly.